x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/session/EsqlResolvedIndexExpressionIT.java

```diff
@@ -37,6 +39,24 @@ public void testLocalPattern() {
         assertThat(resolveIndices("index-*"), equalTo(Map.of(LOCAL_CLUSTER, resolvedIndexExpression("index-*", "index-1,index-2"))));
     }
 
+    // TODO test exclusion pattern
```

Comment:
Could be resolved after https://github.com/elastic/elasticsearch/pull/139230 is merged